drivers/nss/lab01/Makefile

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: GPL-2.0
+
+obj-$(CONFIG_NSS_DRIVERS_LAB01_EX01)	+= # TODO: Ex2 - compile hello.c
+obj-$(CONFIG_NSS_DRIVERS_LAB01_EX04)	+= ex4/hello_name.o
```

Comment:
`TODO: Ex2` is confusing here because they will need to add ``+= ex1/hello.o``. Just just say `TODO: compile hello.c`

Reply:
done